src/cmd/flux-job.c

```diff
@@ -93,8 +93,9 @@ static struct optparse_option list_inactive_opts[] =  {
     { .name = "count", .key = 'c', .has_arg = 1, .arginfo = "N",
       .usage = "Limit output to N jobs",
     },
-    { .name = "timestamp", .key = 't', .has_arg = 1, .arginfo = "T",
-      .usage = "Limit output to jobs newer than timestamp",
+    { .name = "since", .key = 's', .has_arg = 1, .arginfo = "T",
+      .usage = "Limit output to jobs that entered the inactive state since \
+      timestamp T",
```

Comment:
breaking the line in this way results in extra whitespace between `since` and `timestamp`:
```
Usage: flux-job list-inactive [OPTIONS]
List Inactive jobs
  -h, --help             Display this message.
  -c, --count=N          Limit output to N jobs
  -s, --since=T          Limit output to jobs that entered the inactive state
                         since       timestamp T
```

It is actually simple to break strings across multiple lines in C, just create another quoted string on the next line and it will be concatenated:
```C
    { .name = "since", .key = 's', .has_arg = 1, .arginfo = "T",
￼     .usage = "Limit output to jobs that entered the inactive state since"
￼              " timestamp T",
```

Reply:
TIL! Thanks for the clarification @grondo 👍 pushed that change up